package.json

```diff
@@ -46,7 +47,8 @@
     "swagger-router": "^0.0.3",
     "swagger-ui": "git+https://github.com/wikimedia/swagger-ui#master",
     "url-template": "2.0.4",
-    "yargs": "~1.3.0"
+    "yargs": "~1.3.0",
+    "servisor": "git+https://github.com/gwicke/servisor#master"
```

Comment:
Yup. Ideally, lets change straight to a real package once we have decided on a name.